HISTORY.rst

```diff
@@ -61,6 +61,7 @@ Unreleased Changes (3.9)
       a small but noticeable amount of time in launching a model.
     * The number of files included in the python source distribution has been
       reduced to just those needed to install the python package and run tests.
+    * Code-sign the macOS distribution, and switch to a DMG distribution format.
```

Comment:
Might want to mention local doc changes if we decide they should not be bundled with dmg.